Removing xs and *s from seqs...
Modifying peptides...
Calculating peptide physicochemical properties...
../data/kleiner_data/Run1_C4_832ng_rbf_lc-retention-times.csv
Removing xs and *s from seqs...
Modifying peptides...
Calculating peptide physicochemical properties...
../data/kleiner_data/Run1_C4_832ng_linear_lc-retention-times.csv
Removing xs and *s from seqs...
Modifying peptides...
Calculating peptide physicochemical properties...
../data/kleiner_data/Run1_C4_832ng_oligo_lc-retention-times.csv
Removing xs and *s from seqs...
Modifying peptides...
Calculating peptide physicochemical properties...
../data/kleiner_data/Run4_C4_2000ng_rbf_lc-retention-times.csv
Removing xs and *s from seqs...
Modifying peptides...
Calculating peptide physicochemical properties...
../data/kleiner_data/Run4_C4_2000ng_linear_lc-retention-times.csv
Removing xs and *s from seqs...
Modifying peptides...
Calculating peptide physicochemical properties...
../data/kleiner_data/Run4_C4_2000ng_oligo_lc-retention-times.csv
Loading data set...
Traceback (most recent call last):
  File "/home/smccain/miniconda2/envs/pyteo_27/bin/cobia", line 11, in <module>
    load_entry_point('cobia==1.1', 'console_scripts', 'cobia')()
  File "/home-pool/home/smccain/miniconda2/envs/pyteo_27/lib/python2.7/site-packages/cobia-1.1-py2.7.egg/cobia/cobia.py", line 88, in main
    cobia_cofrag(lcfilename = args.lcfilename, ddaparams = args.ddaparams, globalcofrag = args.globalcofrag, output_name = args.output_name, target_df = args.target_df)
  File "/home-pool/home/smccain/miniconda2/envs/pyteo_27/lib/python2.7/site-packages/cobia-1.1-py2.7.egg/cobia/cobia_cofrag.py", line 69, in cobia_cofrag
    elif math.isnan(element_i):
NameError: global name 'math' is not defined
Loading data set...
Traceback (most recent call last):
  File "/home/smccain/miniconda2/envs/pyteo_27/bin/cobia", line 11, in <module>
    load_entry_point('cobia==1.1', 'console_scripts', 'cobia')()
  File "/home-pool/home/smccain/miniconda2/envs/pyteo_27/lib/python2.7/site-packages/cobia-1.1-py2.7.egg/cobia/cobia.py", line 88, in main
    cobia_cofrag(lcfilename = args.lcfilename, ddaparams = args.ddaparams, globalcofrag = args.globalcofrag, output_name = args.output_name, target_df = args.target_df)
  File "/home-pool/home/smccain/miniconda2/envs/pyteo_27/lib/python2.7/site-packages/cobia-1.1-py2.7.egg/cobia/cobia_cofrag.py", line 69, in cobia_cofrag
    elif math.isnan(element_i):
NameError: global name 'math' is not defined
Loading data set...
Traceback (most recent call last):
  File "/home/smccain/miniconda2/envs/pyteo_27/bin/cobia", line 11, in <module>
    load_entry_point('cobia==1.1', 'console_scripts', 'cobia')()
  File "/home-pool/home/smccain/miniconda2/envs/pyteo_27/lib/python2.7/site-packages/cobia-1.1-py2.7.egg/cobia/cobia.py", line 88, in main
    cobia_cofrag(lcfilename = args.lcfilename, ddaparams = args.ddaparams, globalcofrag = args.globalcofrag, output_name = args.output_name, target_df = args.target_df)
  File "/home-pool/home/smccain/miniconda2/envs/pyteo_27/lib/python2.7/site-packages/cobia-1.1-py2.7.egg/cobia/cobia_cofrag.py", line 69, in cobia_cofrag
    elif math.isnan(element_i):
NameError: global name 'math' is not defined
Loading data set...
Traceback (most recent call last):
  File "/home/smccain/miniconda2/envs/pyteo_27/bin/cobia", line 11, in <module>
    load_entry_point('cobia==1.1', 'console_scripts', 'cobia')()
  File "/home-pool/home/smccain/miniconda2/envs/pyteo_27/lib/python2.7/site-packages/cobia-1.1-py2.7.egg/cobia/cobia.py", line 88, in main
    cobia_cofrag(lcfilename = args.lcfilename, ddaparams = args.ddaparams, globalcofrag = args.globalcofrag, output_name = args.output_name, target_df = args.target_df)
  File "/home-pool/home/smccain/miniconda2/envs/pyteo_27/lib/python2.7/site-packages/cobia-1.1-py2.7.egg/cobia/cobia_cofrag.py", line 69, in cobia_cofrag
    elif math.isnan(element_i):
NameError: global name 'math' is not defined
Loading data set...
Traceback (most recent call last):
  File "/home/smccain/miniconda2/envs/pyteo_27/bin/cobia", line 11, in <module>
    load_entry_point('cobia==1.1', 'console_scripts', 'cobia')()
  File "/home-pool/home/smccain/miniconda2/envs/pyteo_27/lib/python2.7/site-packages/cobia-1.1-py2.7.egg/cobia/cobia.py", line 88, in main
    cobia_cofrag(lcfilename = args.lcfilename, ddaparams = args.ddaparams, globalcofrag = args.globalcofrag, output_name = args.output_name, target_df = args.target_df)
  File "/home-pool/home/smccain/miniconda2/envs/pyteo_27/lib/python2.7/site-packages/cobia-1.1-py2.7.egg/cobia/cobia_cofrag.py", line 69, in cobia_cofrag
    elif math.isnan(element_i):
NameError: global name 'math' is not defined
Loading data set...
Traceback (most recent call last):
  File "/home/smccain/miniconda2/envs/pyteo_27/bin/cobia", line 11, in <module>
    load_entry_point('cobia==1.1', 'console_scripts', 'cobia')()
  File "/home-pool/home/smccain/miniconda2/envs/pyteo_27/lib/python2.7/site-packages/cobia-1.1-py2.7.egg/cobia/cobia.py", line 88, in main
    cobia_cofrag(lcfilename = args.lcfilename, ddaparams = args.ddaparams, globalcofrag = args.globalcofrag, output_name = args.output_name, target_df = args.target_df)
  File "/home-pool/home/smccain/miniconda2/envs/pyteo_27/lib/python2.7/site-packages/cobia-1.1-py2.7.egg/cobia/cobia_cofrag.py", line 69, in cobia_cofrag
    elif math.isnan(element_i):
NameError: global name 'math' is not defined
